src/gateway/routers/Aggregator.sol

```diff
@@ -82,12 +82,13 @@ contract Aggregator is Auth, IAggregator {
     // --- Incoming ---
     /// @inheritdoc IAggregator
     function handle(bytes calldata payload) external {
-        Router memory router = activeRouters[msg.sender];
-        require(router.id != 0, "Aggregator/invalid-router");
-        _handle(payload, msg.sender, router, false);
+        _handle(payload, msg.sender, false);
     }
 
-    function _handle(bytes calldata payload, address routerAddr, Router memory router, bool isRecovery) internal {
+    function _handle(bytes calldata payload, address router_, bool isRecovery) internal {
+        Router memory router = activeRouters[router_];
```

Comment:
Realized we had this check in `handle` and `executeMessageRecovery`, both of which call `_handle`, so we could just move it